doc/nrf/protocols/bt/bt_mesh/configuring.rst

```diff
@@ -246,3 +246,44 @@ Using the :ref:`bluetooth_mesh_sensor_server` sample as an example, configured a
   * Ambient light level gain
 
 Adding up all entries, it is worth setting the cache size to minimum 71.
+
+Security toolbox
+----------------
+
+Zephyr's Mesh security toolbox implementation does not include encryption and authentication functionality, such as CMAC, AES-CCM, and HMAC-SHA-256.
+The third-party crypto solutions are used instead.
+
+* The following options are available:
+
+  * :kconfig:option:`CONFIG_BT_MESH_USES_MBEDTLS_PSA` - Enables use of the Mbed TLS PSA API based security toolbox (default option).
+  * :kconfig:option:`CONFIG_BT_MESH_USES_TFM_PSA` - Enables use of the TF-M PSA API based security toolbox (default option for platforms that support TF-M).
+  * :kconfig:option:`CONFIG_BT_MESH_USES_TINYCRYPT` - Enables use of Tinycrypt-based security toolbox.
+    Zephyr's Mesh operates with open value of keys, including storing them in the persistent memory.
+    The Tinycrypt-based solution has worse security materials protection compared to others and not recommended for future designs.
+
+The Bluetooth Mesh security toolbox based on the PSA API does not operate with open key values.
```

Comment:
I think it is worth to use "key values in RAM' instead of saying "open key values"?

Reply:
but it keeps keys value in Flash and RRAM too. I would prefer to follow this formulation "open key value". It will help misinterpretation due to different memory type.